src/web/app/pages-instructor/instructor-audit-logs-page/instructor-audit-logs-page.component.spec.ts

```diff
@@ -243,11 +243,11 @@ describe('InstructorAuditLogsPageComponent', () => {
     expect(studentSpy).toHaveBeenNthCalledWith(2, { courseId: testCourse2.courseId });
   });
 
-  it('should search for logs when search button is clicked', () => {
+  it('should search for logs using feedback course timezone when search button is clicked', () => {
     const logSpy: Spy = spyOn(logService, 'searchFeedbackSessionLog').and
         .returnValue(of({ feedbackSessionLogs: [testLogs1, testLogs2] }));
-    const timeSpy: Spy = spyOn(timezoneService, 'resolveLocalDateTime').and
-        .returnValue(0);
+    const timeSpy: Spy = spyOn(timezoneService, 'resolveLocalDateTime').and.callThrough();
+    const tzOffset: number = timezoneService.getTzOffsets()['Asia/Singapore'];
```

Comment:
If I understand this right, `Asia/Singapore` should not be a magic string but rather a field from `testCourse1`?

Reply:
Right indeed, I've changed it to reference `testCourse1.timezone` instead